packages/aws-cdk-lib/aws-codebuild/test/linux-arm-build-image.test.ts

```diff
@@ -38,17 +38,22 @@ describe('Linux ARM build image', () => {
       });
     });
 
-    test('cannot be used in conjunction with ComputeType MEDIUM', () => {
+    test('can be used with ComputeType MEDIUM', () => {
```

Comment:
Although AMAZON_LINUX_2_STANDARD_1_0 is deprecated, there are tests implemented for STANDARD_1_0, so I decided to keep it as I considered removing it inappropriate. 
If you think it would be better to remove it, please let me know.